DELIVERY.README.txt

```diff
@@ -47,18 +57,26 @@ Each sample will contain the following subfolders:
 
 ProjectID -> SampleN -> 00-Reports
 --------------------------------------------------------------------
-Contains three type of reports, version_report.txt, snpEff summary
-and sample report.
+Contains two type of reports, snpEff summary and sample report.
 
---version_report.txt
-Information from piper, about data sources and software version
-that have been used.
+|--ProjectID
+   |--SampleN
+      |--00-Report
+         |--SampleID_ign_sample_report.html
+         |--SampleID_ign_sample_report.pdf
```

Comment:
you can remove the pdf version of the report, we decided to go for the html version